python_ta/transforms/type_inference_visitor.py

```diff
@@ -275,7 +280,7 @@ def visit_compare(self, node):
                                                             left_value.type_constraints.type,
                                                             right_value.type_constraints.type)
                 left_value = right_value
-                return_types.add(self.type_constraints.unify_call(function_type, left_value.type_constraints.type,
+                return_types.add(self.type_constraints.unify_call(node, function_type, left_value.type_constraints.type,
```

Comment:
I cannot do this because I am using positional arguments in the parameters of unify_call.